cmake/ctest/drivers/atdm/README.md

```diff
@@ -464,18 +464,10 @@ there is no tractability for changes in these settings!
 ## Specific <system_name> directories
 
 The following `<system_name>` sub-directories exist (in alphabetical order):
- 
+
 * `cee-rhel6/`: Contains files to drive builds on CEE LAnL RHEL6 machines with
   a SEMS environment.
 
-* `mutrino/`: Contains files to drive builds on SNL machine mutrino.
-
-* `ride/`: Contains the files to drive builds on the SRN test bed machine
-  `ride` which also can be run on the SON machine `white`.
- 
-* `sems-rhel6/`: Contains files to drive builds on rhel6 machines with the SEMS
-  environment.
-
 * `sems_gcc-7.2.0/`: Contains driver scripts for an on-off GCC 7.2.0 build
   based on the SEMS system.  This build really does not fit into the system
   described above but it put in this directory since it is targeted to support
```

Comment:
This should really be removed too.  This has not been used in a long time.

Reply:
I'm not seeing a `sems_gcc-7.2.0` directory here.  Was it perhaps removed long ago, and we just need to remove the corresponding lines in the `README.md`?